src/axolotl/core/trainer_builder.py

```diff
@@ -1522,6 +1522,8 @@ def build_training_arguments(self, total_num_steps):
         if self.cfg.rl == "orpo":
             training_args_cls = ORPOConfig
             training_args_kwargs["dataset_num_proc"] = self.cfg.dataset_processes
+            training_args_kwargs["max_length"] = self.cfg.sequence_len
+            training_args_kwargs["max_prompt_length"] = self.cfg.sequence_len // 4
```

Comment:
I don't know that we want to artificially limit this to 25% of the sequence len. Either it should be equal to max_length or configurable.

Reply:
The reason behind it was that the ORPO trainer would hang in some of my tests. But having it configurable is fine.